### PR TITLE
rtptools: update to 1.22

### PR DESCRIPTION
**Pull request by janstary**
This updates RTP tools to the latest release.
Two main changes: BSD license and manpages.

Tested on 10.13

### CODE REVIEW

**Comment by janstary** [comment]
Also tested on 10.5.8

File: audio/rtptools/Portfile [hunk 1]
```diff
@@ -3,10 +3,10 @@
 PortSystem          1.0
 
 name                rtptools
-version             1.21
+version             1.22
 categories          audio net
-license             Noncommercial BSD
-platforms           darwin
+license             BSD
+platforms           darwin openbsd linux solaris
```
Comment (l2dy) [inline audio/rtptools/Portfile]: >Possible values: darwin (= macosx or puredarwin), macosx, puredarwin, freebsd, linux, sunos, netbsd. In general, it can just be set to darwin. (puredarwin is an OS based on Apple's open-source Darwin releases without any of Apple's proprietary bits.) See also os.platform.

Why did you add `openbsd` and `solaris`? They are not on [our guide](https://guide.macports.org/#reference.keywords).

Comment (janstary) [reply audio/rtptools/Portfile]: tweaked and squashed
